asati/cli.py

```diff
@@ -20,7 +24,67 @@ def main(*, verbose: bool) -> None:
     logger.info(configure_sentry())
     logger.info("Running process")
 
-    # Do things here!
+    ssm_client = SSMClient()
+    last_accession_parameter = os.environ["LAST_ACCESSION_PARAMETER"]
+    last_accession_uri_id = ssm_client.get_parameter(last_accession_parameter)
+    logger.info(
+        f"SSM parameter '{last_accession_parameter}' retrieved: '{last_accession_uri_id}'"
+    )
+
+    asnake_client = ASnakeClient(
+        baseurl=os.environ["ARCHIVESSPACE_URL"],
+        username=os.environ["ARCHIVESSPACE_USER"],
+        password=os.environ["ARCHIVESSPACE_PASSWORD"],
+    )
+    accession_uri_ids = asnake_client.get(
+        "/repositories/2/accessions?all_ids=true"
+    ).json()
+
+    if max(accession_uri_ids) <= int(last_accession_uri_id):
+        logger.info("No new accessions to add to Airtable.")
+    else:
```

Comment:
Instead of a nested `if/else` here, where _most_ of the meaningful code is under `else`, I'd propose exiting early if there are no accessions to process.  That would allow for de-denting the bulk of this code to the "default" path unless it bailed early.

Side note: I think it's okay if we don't print the elapsed time given an early exit, or could duplicate that logging code.

Reply:
That is the right call and I don't think the elapsed time is necessary for a `No new accessions` run. Great suggestion, thanks!